recipes/hug/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "hug" %}
+{% set version = "2.1.2" %}
+{% set sha256 = "a9bd00e14549001d89b4cff08742d9da50f2f6a1fb0cf9c872bfb405e66d0a51" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/timothycrosley/hug/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - hug = hug:development_runner.hug.interface.cli
+
+requirements:
+  build:
+    - python 3.*
+    - setuptools
+    - falcon ==1.0.0
+    - requests
+
+  run:
+    - python 3.*
+    - falcon ==1.0.0
+    - requests
+
+test:
+  imports:
+    - hug
+
+  commands:
+    - hug --help
+
+  requires:
+    - mock
+    - pytest
+
+about:
+  home: http://www.hug.rest/
+  license: MIT License
```

Comment:
Please shorten to `MIT`. Also would recommend adding `license_family: MIT`.

Also please add `license_file: LICENSE`.